deployment/v3/docs/external services configuration.md

```diff
@@ -0,0 +1,82 @@
+#  External Services Configuration
+
+* This document describes how to configure external services like Minio,Keycloak, and Postgres
+
+
+## Postgres external configuration procedure
+
+* Go to mosip-config and checkout to a specific branch to update config properties.
```

Comment:
* Go to `mosip-config` and checkout to a specific branch to update the `config` properties.